reference/api/tasks.md

```diff
@@ -6,99 +6,98 @@ The `/tasks` route gives information about the progress of [asynchronous operati
 The task `uid` is incremented **globally.**
 :::
 
-## Get all tasks
+## Get tasks
```

Comment:
Would it be a good idea if we set the example headings to H4 and add `sidebarDepth: 2`?That way the filtering and paginating headings will be more visible

<img width="247" alt="Screen Shot 2022-06-21 at 14 48 06" src="https://user-images.githubusercontent.com/90181761/174782229-8cac1274-0ef1-4c06-88fa-2b8ec92536fd.png">

Reply:
I also like the idea of highlighting subsections that might help users, and I also agree it doesn't belong to this PR.

Something to take into account, though: HTML headings (h1 through h6) are semantic structures. As in, using an H4 after an H3 implies the H4 is a subsection of H3. I also _think_ that not using headings sequentially (e.g. H1 followed by H3 instead of H1 followed by H2) can mess up the [document outline](https://www.danielfranklin.id.au/understanding-document-outline-in-html/). 

What I'm trying to say is: we should avoid as much as possible using a specific heading **only** because of the effect this will have on the sidebar. AFAIK, this is particularly relevant when it comes to accessibility because screen readers rely on the document outline to help visually impaired users navigate a page. If we find ourselves willingly using tags erroneously, it's an indication we need to update our website/tooling/etc.